cfgov/jinja2/v1/_includes/organisms/data_snapshot.html

```diff
@@ -50,7 +50,9 @@ <h3><b>{{ value.year_over_year_change }}</b>
         <h3><b>{{ value.inquiry_year_over_year_change }}</b>
         <br/>
         {{ value.inquiry_year_over_year_change_text }}
-        ({{ value.inquiry_month.strftime('%B&nbsp;%Y') }})</h3>
+        {% if value.inquiry_month %}
+            ({{ value.inquiry_month.strftime('%B&nbsp;%Y') }})</h3>
```

Comment:
This change puts the closing `</h3>` tag inside the `{% if %}` clause, so that if the condition is false, the above `<h3>` will never be closed.

Reply:
Good catch, I'm fixing it now.